manifests/s/SmartGameBooster/SmartGameBooster5/2/5.2.0.567/SmartGameBooster.SmartGameBooster5.2.yaml

```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: SmartGameBooster.SmartGameBooster5.2
+PackageVersion: 5.2.0.567

```

Comment:
```suggestion
PackageIdentifier: SmartGameBooster.SmartGameBooster
PackageVersion: 5.2.0
```

Reply:
I have done these changes in #27970
This should work